sdk/servicebus/service-bus/samples/typescript/package.json

```diff
@@ -30,7 +30,7 @@
   "dependencies": {
     "@azure/abort-controller": "^1.0.0",
     "@azure/identity": "^1.0.2",
-    "@azure/service-bus": "next",
+    "@azure/service-bus": "file:../../azure-service-bus-7.0.0-preview.5.tgz",
```

Comment:
This should be reverted before we merge?

Reply:
Ouch

Reply:
Reverted